### DISCUSSION TITLE
2 expo apps 1 web app

### DISCUSSION THREAD

**Discussion by karmatradeDev** [nandorojo/solito#297] [Q&A]
hi question can i do 2 expo apps in solito? example doordash app and doordash driver app and doordash website

**Comment by albertcito** [comment]
Looks like Expo has some manual in order to add more apps in the same repo:

https://docs.expo.dev/guides/monorepos/

**Comment by nandorojo** [comment]
Yes you can. I would probably copy the source of `apps/expo` (without node modules) and then change the app config file
